dev/TabView/TabView.cpp

```diff
@@ -504,15 +613,120 @@ winrt::DependencyObject TabView::ContainerFromIndex(int index)
     return nullptr;
 }
 
-void TabView::OnCtrlF4Invoked(const winrt::KeyboardAccelerator& sender, const winrt::KeyboardAcceleratorInvokedEventArgs& args)
+int TabView::GetItemCount()
+{
+    if (auto itemssource = ItemsSource())
+    {
+        return winrt::make<InspectingDataSource>(ItemsSource()).Count();
+    }
+    else
+    {
+        return static_cast<int>(Items().Size());
+    }
+}
+
+bool TabView::SelectNextTab()
```

Comment:
Can you make SelectNextTab(int increment) and get rid of SelectPreviousTab? The wraparound logic can be handled either direction with:
```
    index = (index + increment + itemsSize) % itemsSize;
```

Reply:
Done.